src/utils/speech_to_text_api.py

```diff
@@ -0,0 +1,38 @@
+import os
+import io
+
+from google.cloud import speech
+
+class SpeechToTextAPI:
+
+        def __init__(self):
+            pass
+
+        @staticmethod
+        def resquestTranscription(serviceAccountPath, audioDataPath, encoding, sampleRate, languageCode, model, enhanced):
```

Comment:
I think this is a lot of arguments, we should have like 3-4 arguments and the last one is a "options" object. Like :

    requestTranscription(serviceAccountPath, audioDataPath, options={'encoding' : 'utf-8', etc})

What I did here is a definition of a default value for options, in the case the options argument isn't pass when it is use.